frontend/src/pages/TrainingReports/components/EventCard.js

```diff
@@ -73,6 +84,26 @@ function EventCard({
           )}
         </div>
       </div>
+
+      <div className="margin-top-3">
+        <ExpanderButton
+          type="report"

```

Comment:
should be "session" I think, the mockup says "hide sessions" and "view sessions"

Reply:
done.